src/SFA.DAS.Commitments.Domain/Entities/Relationship.cs

```diff
@@ -7,7 +7,7 @@ public class Relationship
         public string LegalEntityId { get; set; }
         public string LegalEntityName { get; set; }
         public string LegalEntityAddress { get; set; }
-        public OrganisationType LegalEntityOrganisationType { get; set; }
+        public SFA.DAS.Common.Domain.Types.OrganisationType LegalEntityOrganisationType { get; set; }
```

Comment:
This is explicitly named here to avoid accidental and contamination with the still available, but now obsolete, local enum of the same name